### PR TITLE
T cycle

### PR DESCRIPTION
**Pull request by vuonghtruong**
added analysis

### CODE REVIEW

**Review by JihwanMyung — APPROVED** [review]

File: BTLocoBox_0003_v3/dist/BTLocoBox_0003_v3_Box1-5.exe
Comment (JihwanMyung) [inline BTLocoBox_0003_v3/dist/BTLocoBox_0003_v3_Box1-5.exe:1]: Somehow I am unable to resolve the conflict caused by large exe..

**Comment by vuonghtruong** [comment]
Yes, the conflict is just “deleting the exe files” as we do not have enough storage on GitHub (hence, you can just ignore it). We might want to upload the exe file elsewhere (your dropbox, google drive) and include the shared link to README.mdI have uploaded the two exe files to our Braintime disk at /home/Drive/Software/BTLocoBox_003_v3  From: Jihwan MyungSent: Monday, October 16, 2023 5:43 PMTo: JihwanMyung/LocoBoxCc: Vuong Truong; AuthorSubject: Re: [JihwanMyung/LocoBox] T cycle (PR ***@***.*** commented on this pull request.On BTLocoBox_0003_v3/dist/BTLocoBox_0003_v3_Box1-5.exe:Somehow I am unable to resolve the conflict caused by large exe..—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

File: BTLocoBox_0003_v3/dist/BTLocoBox_0003_v3_Box1-5.exe
Comment (vuonghtruong) [reply BTLocoBox_0003_v3/dist/BTLocoBox_0003_v3_Box1-5.exe:1]: Yes, the conflict is just “deleting the exe files” as we do not have enough storage on GitHub (hence, you can just ignore it). We might want to upload the exe file elsewhere (your dropbox, google drive) and include the shared link to README.md
I have uploaded the two exe files to our Braintime disk at /home/Drive/Software/BTLocoBox_003_v3

**Comment by JihwanMyung** [comment]
I probably messed up somewhere while rebasing... I cannot pull and merge. Can you push again?

jihwan@JM-MBP14 BTLocobox % git pull origin master
From https://github.com/JihwanMyung/LocoBox
 * branch            master     -> FETCH_HEAD
HEAD is up to date.
jihwan@JM-MBP14 BTLocobox % git checkout T_cycle
error: pathspec 'T_cycle' did not match any file(s) known to git
jihwan@JM-MBP14 BTLocobox % git merge master
Already up to date.
jihwan@JM-MBP14 BTLocobox % ls
1LED_1PIR_10Boxes_Tcycle	BTLocobox_0001_v1		RTC_sim
BTLocoBox_0002_v2		LocoBox_12Phase			test_unix_time
BTLocoBox_0003_v3		README.md
BTLocobox_0000_v0		RTC_DS1307-master.zip

**Pull request closed without merging**